firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/ImagenInlineImage.kt

```diff
@@ -1,12 +1,37 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package com.google.firebase.vertexai.type
 
 import android.graphics.Bitmap
 import android.graphics.BitmapFactory
 import android.util.Base64
 
+/**
+ * Represents an Imagen Image that is contained inline
```

Comment:
What is "Imagen Image"?

Reply:
A relic of an older API that i'd forgotten to delete.

Reply:
Will you be deleting this in a separate PR?